src/main/java/io/javaalmanac/snippets/util/OptionalValues.java

```diff
@@ -8,7 +8,7 @@
  * to express the possibility that a method may not return a value, for example
  * {@link java.util.stream.Stream#max(java.util.Comparator)} may not have a
  * return value on an empty stream.
- * 
+ * <p>
```

Comment:
Same here: We need Markdown.

Reply:
sorry, I didn't know about the rendering pipeline. I've also fixed javadoc in the second PR ttps://github.com/marchof/io.javaalmanac.snippets/pull/4